addfp_5110.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+python /home/mango/goodix-fp-dump/run_5110.py
```

Comment:
What about just using `python run_5110.py`. This script should be run in `goodix-fp-dump` anyway. This will fail for any other user, so can you review this? Thanks.